pyega3/pyega3.py

```diff
@@ -143,7 +143,7 @@ def main():
     auth_client.credentials = credentials
 
     data_client = DataClient(server_config.url_api, server_config.url_api_ticket, auth_client, standard_headers,
-                             connections=args.connections)
+                             connections=args.connections, metadata_url=server_config. url_api_metadata)
```

Comment:
I sent the suggestion with a weird space
```suggestion
                             connections=args.connections, metadata_url=server_config.url_api_metadata)
```